charts/alfresco-repository/templates/secret-message-broker.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if and (not .Values.configuration.messageBroker.existingSecret.name) .Values.configuration.messageBroker.username .Values.configuration.messageBroker.password }}
```

Comment:
may not be of high impact, but the upstream activemq doesn't have any credentials enabled by default (or at least that what I remember), maybe we are too strict here? in the end generating a secret with empty credentials is still valid

Reply:
Not generating the secret at all will result in the same outcome: empty variables si that will work with unauthenticated activemq.

Reply:
true, and I see now the `optional: true` on the ref so it's cool